recipes/tcsh/meta.yaml

```diff
@@ -0,0 +1,37 @@
+{% set name = "tcsh" %}
+{% set version = "6.20.00" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  git_url: https://github.com/tcsh-org/tcsh.git
+  git_tag: TCSH6_20_00
```

Comment:
We prefer not to clone from version control. Is there a tarball that we can use instead?

Reply:
tcsh.org still seems to be down as commented in the OP, but https://github.com/tcsh-org/tcsh/releases should work.

Reply:
...and tcsh.org is still down...don't know if the project is even still active. Since the github repo is tagged I'm hoping this will be sufficiently stable...

Reply:
@jamesabbott, please still change it to `url: https://github.com/tcsh-org/tcsh/archive/TCSH{{ version.replace('.', '_') }}.tar.gz`.